.env.sample

```diff
@@ -8,6 +8,7 @@ UNIVERSAL_ANALYTICS_GTM_CONTAINER_ID="GTM-TK92W68"
 GOOGLE_ANALYTICS_4_GTM_CONTAINER_ID="GTM-KD86CMZ"
 ANALYTICS_DOMAIN="localhost"
 LOG_LEVEL="debug"
+LANGUAGE_TOGGLE_ENABLED=0
```

Comment:
Why are we using 0 (and I assume 1?) rather than true/false?

Reply:
It's an excellent question.
Because other env variable related to "language" we have found is taking the value 0 / 1
It's used in several frontend repo and named SUPPORT_LANGUAGE_CY. 
So for keeping some consistency, we used the same pattern.

Reply:
Hmm have you taken into consideration the other feature toggles like `GA4_DISABLED` and `UA_DISABLED` though? These are booleans. 

Does `showLanguageToggle` have to be 0/1 or does it just need to be truthy/falsy? I can't see anything mentioned in the common express or language toggle docs

Reply:
It needs to be 0 or 1
Related function into common express:
https://github.com/govuk-one-login/ipv-cri-common-express/blob/main/src/lib/locals.js

Reply:
Cool I've updated the PR so that the env var matches our pattern of env vars but there is transformation done to fit in with common express